test/CMakeLists.txt

```diff
@@ -1,429 +1,436 @@
-    gct
-    gctrl
-    gd
-    gddot
-    gdll
-    gdll_df
-    ge
-    gebar
-    gesc
-    gf
-    gfor
-    gft
-    ggoto
-    gh
-    gi
-    gi0
-    gibs
-    gibst
-    gicap
-    gicap2
-    gico
-    giconv
-    gif
-    gimaxmin
-    gintdiv
-    gintg
-    gintovfl
-    giph
-    gipht
-    giscode
-    git
-    gix
-    gj
-    glco
-    gldot
-    glocale
-    gmean
-    gmemo
-    # gmmf bus errors, unstable
-    # gmmf1s 
-    # gmmf1u
-    # gmmf1w
-    gmnom
-    gn
-    gnan
-    gnum
-    gnvv
-    go
-    goi
-    goox
-    gop
-    gos
-    gparse
-    gpco
-    gpco2
-    gpcoinv
-    gpdd
-    gpi
-    gpick
-    gpoly
-    gq
-    gq101
-    gq132
-    gq201
-    gqco
-    gqnonrat
-    gr
-    gr1
-    grefcount
-    gs
-    gsco
-    gsco1
-    gsco1u
-    gsco1w
-    gsco2
-    gsco2u
-    gsco2w
-    gscou
-    gsdot
-    gselect
-    gsp
-    gsp0
-    gsp000
-    gsp1
-    gsp100
-    gsp101
-    gsp102
-    gsp110
-    gsp111
-    gsp112
-    gsp120
-    gsp122
-    gsp130
-    gsp2
-    gsp221
-    gsp222
-    gsp231
-    gsp320
-    gsp321
-    gsp322
-    gsp331
-    gsp400
-    gsp410
-    gsp412
-    gsp420
-    gsp421
-    gsp422
-    gsp431
-    gsp432
-    gsp520
-    gsp520sd
-    gsp520ss
-    gsp521
-    gsp530i
-    gsp530l
-    gsp530n
-    gsp531
-    gsp5x5
-    gsp6
-    gsp600
-    gsp600a
-    gspi
-    gspj
-    gspo
-    gspr
-    gspx
-    gss
-    gstack
-    gt
-    gthrow
-    gtrain
-    gtry
-    gu
-    gu0
-    gunderai
-    gwhile
-    gx
-    gx132
-    gxco
-    gxco1
-    gxco2
-    gxinf
-)
-
-foreach(test_name IN LISTS test_cases)
+            ../jsrc/tsdll.c
+            )
+endif ()
+set(fast_test_cases
+        g0
+        g000
+        g000a
+        g000i
+        g000p
+        g000s
+        g001
+        g010
+        g010a
+        g010i
+        g010p
+        g010s
+        g011
+        g011a
+        g011i
+        g011p
+        g011s
+        g012
+        g012a
+        g012i
+        g012p
+        g012s
+        g020
+        g020a
+        g020i
+        g020p
+        g020s
+        g021
+        g021a
+        g021i
+        g021p
+        g021s
+        g022
+        g022a
+        g022i
+        g022p
+        g022s
+        g030
+        g030a
+        g031
+        g032
+        g0a
+        g0x
+        g100
+        g100a
+        g100i
+        g100p
+        g100s
+        g101
+        g101a
+        g101i
+        g101p
+        g101s
+        g102
+        g102a
+        g102i
+        g102p
+        g102s
+        g110
+        g110a
+        g110i
+        g110p
+        g110s
+        g111
+        g111a
+        g111i
+        g111p
+        g111s
+        g112
+        g112a
+        g112i
+        g112p
+        g112s
+        g120
+        g120a
+        g120i
+        g120p
+        g120s
+        g121
+        g122
+        g128x
+        g128x3
+        g128x5
+        g130
+        g130a
+        g130i
+        g130p
+        g131
```

Comment:
Move to slow:
```
30/390 Test  #88: fast_g131 ........................   Passed   15.85 sec
```

Reply:
hmm I got about half the time on my pc. Though I wasn't sure when I first did this where the cut off should be. So you said 5 seconds so since my timing is a bit faster on my pc than the CI i went with 3 second cutoff. I also added the time I recorded as a comment to the cmakelists.txt